packages/admin-stories/src/admin/table/GlobalErrorHandling.tsx

```diff
@@ -0,0 +1,74 @@
+import { Table, TableQuery, useTableQuery } from "@comet/admin";
+import { text } from "@storybook/addon-knobs";
+import { storiesOf } from "@storybook/react";
+import gql from "graphql-tag";
+import * as React from "react";
+
+import { apolloSwapiStoryDecorator } from "../../apollo-swapi-story.decorator";
+import { errorDialogStoryProviderDecorator } from "../../error-dialog-provider.decorator";
+
+interface People {
+    id: string;
+    name: string;
+    birthYear: string;
+    gender: string;
+    homeworld: {
+        name: string;
+    };
+}
+interface QueryData {
+    allPeople: {
+        people: People[];
+    };
+}
+
+const Story: React.FunctionComponent = () => {
+    const query = gql`
+        ${text("GQL Query", "query StarWarsPeople {allPeople { people { id name birthYear gender homeworld{ name } } }}")}
```

Comment:
I took me quite some while to understand that I need to make this story fail by passing an invalid GraphQL query. Isn't there a better way to demonstrate the functionality? 🤔

Reply:
that's the best solution i found to explicitly throw an error. What to you suggest?

Reply:
How about creating a Button that changes a query Variable to an unsupported value?

Reply:
the button idea sounds nice. i will give it a try.

Reply:
good idea!
![Screen Recording 2021-05-19 at 13 14 26 mov](https://user-images.githubusercontent.com/6098356/118803995-6561fd00-b8a4-11eb-8158-e20f0cb704e8.gif)